.github/workflows/trivy-analysis.yml

```diff
@@ -1,63 +1,36 @@
 name: Trivy
```

Comment:
@abstractj can you review this one please?

Reply:
LGTM